pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -960,6 +979,17 @@ func lookupAppNetworkConfig(ctx *zedrouterContext, key string) *types.AppNetwork
 	return &config
 }
 
+func lookupVolumeRefList(ctx *zedrouterContext, key string) []types.DiskStatus {
```

Comment:
It seems the name not match the logic.

Reply:
indeed. Will change to `lookupDiskStatusList`